pkg/agent/route/route.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"net"
+
+	"antrea.io/antrea/pkg/agent/util"
+)
+
+var (
+	// ServiceGWVirtualIP is a virtual IP for redirecting ClusterIP Service traffic from host network to OVS.
+	// The traffic will be routed to antrea-gw0 via the virtual IP. It is added to use a single
+	// neighbor cache entry for the traffic (rather than one entry for each ClusterIP).
+	// TODO: Integrate with hairpinIP related flows
+	ServiceGWVirtualIP    = net.ParseIP("169.254.169.252").To4()
+	// ServiceGWVirtualMAC is a virtual MAC used to match ServiceGWVirtualIP in a neighbor cache entry.
+	ServiceGWVirtualMAC, _ = net.ParseMAC("aa:bb:cc:dd:ee:fe")
```

Comment:
ditto, they are duplicate with the ones for Linux. And I don't think we want different IPs and MACs for Linux and Windows for same purpose virtual IP and MAC.

Reply:
"169.254.169.252" is for arp responder/netneighbor in Windows, which is different from "169.254.169.253".

Reply:
No, please check comments of VirtualServiceIPv4 and the routes in route_linux.go. 
They are exactly same, except the next hop would be 169.254.169.253 on Linux and 169.254.169.252 on Windows.

Reply:
Will try.